src/lib/docs/Utilities/ResourceProvider.md

```diff
@@ -3,7 +3,7 @@ title: Resource Provider Spec
 description: A specification to allow 3rd party resource providers to cosign transactions and cover the resource cost of individual transactions.
 category: Utilities
 published: true
-requiresReview: true
```

Comment:
We need `requiresReview` on all of these. Make sure to include them in your new docs, as its what Mara uses to find them in order to do reviews.